avakas/flavors/git.py

```diff
@@ -68,9 +68,10 @@ def __determine_bump(self):
         vsn = None
         reg = re.compile(r'(\#|bump:|\[)(?P<bump>(patch|minor|major))(.*|\])',
                          re.MULTILINE)
+        tagged_commits = set(tag.commit for tag in self.repo.tags)
         for commit in self.repo.iter_commits(self.options['branch']):
             # we go iterate back to the last time we bumped the version
-            if commit.message.startswith('Version bumped to'):
```

Comment:
I don't think this has any side effects..... iirc even with the legacy `version` file bit everything has a git tag unless explicitly requested otherwise.